src/dataset-builder/CohortEditor.test.ts

```diff
@@ -163,7 +155,7 @@ describe('CohortEditor', () => {
 
   it('renders list criteria', async () => {
     // Arrange
-    mockListStatistics();
+    mockDataRepo({ ...listStatisticsMock(), ...getSnapshotBuilderCountMock() });
```

Comment:
This seems a little awkward, could `mockDataRepo()` accept `[]` and internally convert it before passing it to `mockImplementation()`?

Reply:
I also agree